src/components/UI/FormFields/InputField/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { TextField, TextFieldProps } from "@mui/material";
+import { FieldError } from "../FieldError";
+
+type IProps = {
```

Comment:
```suggestion
interface IProps = {
```

Reply:
Not possible with mui with this type

Reply:
Maybe try FilledTextFieldProps? I think that have a solution with interface instead

Reply:
Thanks it works

Reply:
Fixed